### PR TITLE
Add vp2430 to the test environment

### PR DESCRIPTION
**Pull request by JanPrusinowski**

### CODE REVIEW

**Comment by macpijan** [comment]
Let's wait for some successful test, at least basic-interfaces one.

**Comment by macpijan** [comment]
@JanPrusinowski There is already older PR with this config: https://github.com/Dasharo/open-source-firmware-validation/pull/554

File: platform-configs/protectli-vp2430.robot [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+${CPU_MAX_FREQUENCY}=           3800
+${PLATFORM_RAM_SPEED}=          4800
+${PLATFORM_RAM_SIZE}=           16384
+${WIFI_CARD_UBUNTU}=            Qualcomm Atheros QCA6174 802.11ac Wireless Network Adapter (rev 30)
+${BLUETOOTH_CARD_UBUNTU}=       Qualcomm Atheros Communications AR3012 Bluetooth 4.0
+${E_MMC_NAME}=                  BJTD4R
+${CPU}=                         Intel(R) Core(TM) i3-N305
+${DMIDECODE_MANUFACTURER}=      Protectli
+${DMIDECODE_SERIAL_NUMBER}=     123456789
+${DMIDECODE_PRODUCT_NAME}=      VP3230
```
Comment (miczyg1) [inline platform-configs/protectli-vp2430.robot:19]: 3230?

File: platform-configs/protectli-vp2430.robot [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+${INITIAL_CPU_FREQUENCY}=       800
+${PLATFORM_CPU_SPEED}=          3,80
+${CPU_MIN_FREQUENCY}=           800
+${CPU_MAX_FREQUENCY}=           3800
+${PLATFORM_RAM_SPEED}=          4800
+${PLATFORM_RAM_SIZE}=           16384
+${WIFI_CARD_UBUNTU}=            Qualcomm Atheros QCA6174 802.11ac Wireless Network Adapter (rev 30)
+${BLUETOOTH_CARD_UBUNTU}=       Qualcomm Atheros Communications AR3012 Bluetooth 4.0
+${E_MMC_NAME}=                  BJTD4R
+${CPU}=                         Intel(R) Core(TM) i3-N305
```
Comment (miczyg1) [inline platform-configs/protectli-vp2430.robot:16]: It is `Intel(R) N100`

File: platform-configs/protectli-vp2430.robot [hunk 3]
```diff
@@ -0,0 +1,27 @@
+*** Settings ***
+Resource    include/protectli-vp24xx.robot
+
+
+*** Variables ***
+# Automatically found variables
+${INITIAL_CPU_FREQUENCY}=       800
+${PLATFORM_CPU_SPEED}=          3,80
+${CPU_MIN_FREQUENCY}=           800
+${CPU_MAX_FREQUENCY}=           3800
```
Comment (miczyg1) [inline platform-configs/protectli-vp2430.robot:10]: This needs to be revisited as it seems to be copy-paste

**Review by miczyg1 — CHANGES_REQUESTED** [review]

**Comment by macpijan** [comment]
@philipandag  Consider looking at https://github.com/Dasharo/open-source-firmware-validation/pull/554 - maybe it has something more than present here already: https://github.com/Dasharo/open-source-firmware-validation/pull/554

**Comment by philipandag** [comment]
The platform configs will be added in this PR https://github.com/Dasharo/open-source-firmware-validation/pull/554
This one is not needed anymore

**Pull request closed without merging**